src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -352,6 +352,7 @@ extern bool entitiesQuery
   const std::vector<std::string>&  servicePath,
   int                              offset  = DEFAULT_PAGINATION_OFFSET_INT,
   int                              limit   = DEFAULT_PAGINATION_LIMIT_INT,
+  bool*                            limitReached = NULL,
   long long*                       countP  = NULL
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/9101970658842d30fb0d1a0166d115e5d0358a8c